pandas/core/series.py

```diff
@@ -918,12 +918,49 @@ def _set_values(self, key, value):
     @deprecate_kwarg(old_arg_name='reps', new_arg_name='repeats')
     def repeat(self, repeats, *args, **kwargs):
         """
-        Repeat elements of an Series. Refer to `numpy.ndarray.repeat`
-        for more information about the `repeats` argument.
+        Repeat elements of a Series.
+
+        Each element of the Series is repeated 'repeats' times.
+
+        Parameters
+        ----------
+        repeats : int
+            The number of repetitions for each element.
+        *args
```

Comment:
You can remove args and kwargs. The validation function is an implementation detail that users shouldn't have to worry about. `repeats` is the only valid argument.

You *could* mention, perhaps in `Notes` that "unlike `numpy.repeat`, \`axis\` is not a valid argument".

Reply:
Listen to @jorisvandenbossche about args and kwargs, not me :)

Reply:
After discussion, you can document them like

```
*args, **kwargs
    Additional keywords have no effect but might be accepted for
    compatibility with numpy.
```